docs/source/Getting-Started/index.rst

```diff
@@ -2,11 +2,11 @@ Getting Started
 ==================================
 
 What is Keystone?
------------------------------------
+--------------------------------------------
 
 Keystone is an open-source TEE framework for RISC-V processors.
 
-You can currently run Keystone on qemu, `FireSim <https://fires.im/>`_ (FPGA), or the SiFive `HiFive Unleashed <https://www.sifive.com/boards/hifive-unleashed>`_ board.
```

Comment:
Why try rather than run?

Reply:
Thought was that "Keystone" is a framework, and you can run an enclave not Keystone...